_aggregations/metric/minimum.md

```diff
@@ -9,30 +9,46 @@ redirect_from:
 
 # Minimum aggregations
 
-The `min` metric is a single-value metric aggregations that returns the minimum value of a field.
+The `min` metric is a single-value metric that returns the minimum value of a field.
 
-The following example calculates the minimum of the `taxful_total_price` field:
+The `min` aggregation compares numeric fields using a `double` (double-precision) representation. Results should be considered approximate for fields containing `long` or `unsigned_long` integers with absolute values greater than 2<sup>53</sup> because the number of significant bits in a `double` mantissa is 53.
+{: .note}
+
+## Parameters
+
+The `min` aggregation takes the following parameters.
+
+| Parameter | Required/Optional | Data type      | Description |
+| :--       | :--               | :--            | :--         |
+| `field`   | Required          | String         | The name of the field for which the minimum is computed.    |
+| `missing` | Optional          | Numeric        | The value to assign missing instances of the field. If not given, documents with missing values are omitted from the aggregation. |
+
+## Example
+
+This example finds the least expensive item---the item with the minimum value of the `base_unit_price`---in the e-commerce sample data:
 
 ```json
 GET opensearch_dashboards_sample_data_ecommerce/_search
 {
   "size": 0,
   "aggs": {
-    "min_taxful_total_price": {
+    "min_base_unit_price": {
       "min": {
-        "field": "taxful_total_price"
+        "field": "products.base_unit_price"
       }
     }
   }
 }
 ```
 {% include copy-curl.html %}
 
-#### Example response
+## Example response
+
+The response is as follows:
```

Comment:
@dwelsch-esi Let's rewrite to include more context (what does the response demonstrate to the user?).

Reply:
```suggestion
As shown in the following example response, the aggregation returns the minimum value of `products.base_unit_price`.
```

Reply:
@dwelsch-esi Same

Reply:
```suggestion
As shown in the following example response, the aggregation returns the minimum value of `products.base_unit_price`:
```